src/ansible_compat/runtime.py

```diff
@@ -128,12 +128,7 @@ def install_collection(
             _logger.error("Command returned %s code:\n%s", run.returncode, run.stdout)
             raise InvalidPrerequisiteError()
 
-    @tenacity.retry(  # Retry up to 3 times as galaxy server can return errors
-        reraise=True,
-        wait=tenacity.wait_fixed(30),  # type: ignore
-        stop=tenacity.stop_after_attempt(3),  # type: ignore
-        before_sleep=tenacity.after_log(_logger, logging.WARNING),  # type: ignore
-    )
+    @retry
```

Comment:
Btw the shim could just accept all these args and bypass them conditionally. Not sure if it's important right now. It's probably okay to keep it if this is the only use atm.